sass/components/_announcement-bar.scss

```diff
@@ -57,6 +89,45 @@
     .resend-verification-wrapper:before {
         content: "\00a0 ";
     }
+
+    .content__icon {
+        left: -6.25%;
+        right: -6.25%;
+        top: -4.38%;
+        bottom: -8.12%;
```

Comment:
remove left, right, bottom positioning

change top to `top: 1px`
add `position: relative`